Data/DataModel/Prerequisites.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ServerCore.DataModel
+{
+    public class Prerequisites
```

Comment:
Can you add comments to this? I had to stop and think about what a prerequisite would mean and what this is, which makes a good candidate for comments. Xml comments are best since then Intellisense knows all the things. (I'm aware that a lot of the data model doesn't have comments yet, it's a work in progress.)

Reply:
Sure, will do.